addons/binding/org.openhab.binding.netatmo/README.md

```diff
@@ -92,6 +92,10 @@ Number Netatmo_Indoor_CO2 "CO2" <carbondioxide> { channel = "netatmo:NAMain:home
 * Location
 * TimeStamp
 * LastStatusStore
+* MinTemp
+* MaxTemp
```

Comment:
I can confirm that all new channels work for the NAMain and the NAModule1. I asked in the [community](https://community.openhab.org/t/what-channel-types-are-supported-by-wich-netatmo-device/24394/19?u=cweitkamp) if anybody owns a NAModule4 for testing.